server/test/printing.go

```diff
@@ -0,0 +1,14 @@
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func PrettyPrintJSON(prefix string, v any) {
+	bytes, err := json.MarshalIndent(v, "", "  ")
+	if err != nil {
+		fmt.Printf("pretty-print JSON error: %v\n", err)
+	}
+	fmt.Println(prefix + ": " + string(bytes))
+}
```

Comment:
¿Qué es?

Reply:
Just a debugging function

Reply:
ya I just didn't see it used anywhere

Reply:
I removed them because I finished debugging hehe

Reply:
Gabe and Jahziel were talking in slack a couple of days ago about how we don't have a simple function that does this so I just added it, assuming people will use it